src/components/checkbox/checkbox.scss

```diff
@@ -39,6 +39,20 @@
                 }
```

Comment:
I'll assign myself and put this on the Feature team's cycle board too, so it's visible to my team that I'm reviewing this, and so that I don't forget 😄